File: wwwroot/Scripts/PersonaTS/Persona.ts
import {Usuario} from '../UsuarioTS/Usuario';

class Persona
{
    public item : boolean;
    public id : number;
    public localidadId : number;
    public provinciaId : number;
    public nombre : string;
    public apellido : string;
    public dni : number;
    public fechaNacimiento : Date;
    public email : string = null;
    public celular : number;
    public telefono : number = null;
    public usuario : Usuario = null;
    // SOLO LECTURA
    public apyNom : string;
    public localidad : string;
    public fechaStr : string;
}

export {Persona};

File: Entidades/Persona.cs
namespace Entidades
{
    using Entidad.Base;
    using System;
    public class Persona : EntityBase
    {
        public long LocalidadId { get; set; }
        public string Nombre { get; set; }
        public string Apellido { get; set; }
        public int Dni { get; set; }
        public DateTime FechaNacimiento { get; set; }
        #nullable disable
        public string Email { get; set; }
        #nullable disable
        public int Celular { get; set; }
        public int? Telefono { get; set; }

        // Navegacion
        public virtual Localidad Localidad { get; set; }
        public virtual Usuario Usuario { get; set; }
    }
}